src/book/03-guides/01-main-codebase/01-getting-started.mdx

```diff
@@ -359,7 +359,7 @@ Open up a new terminal to run NUsight and NUbots code side by side.
 2. Install Node.js by running
 
    ```bash
-   sudo apt install node
+   sudo apt install npm nodejs
```

Comment:
Ubuntu's default `nodejs` is usually a few LTS versions behind. For example, the current version on Ubuntu 20.04 is Node 10, when the latest LTS is 14, soon to be 16.

I normally point people to https://github.com/nodesource/distributions/blob/master/README.md to and tell them to install from the section with the latest LTS, which at the time of writing is 14. These instructions will do:

```suggestion
   curl -fsSL https://deb.nodesource.com/setup_14.x | sudo -E bash -
   sudo apt-get install -y nodejs
```

Reply:
Does this take care of the npm issue too? Or do we need npm still in here?

Reply:
Still need npm, and the default Ubuntu version is OK. Maybe move it after this step, before the yarn step.